rest-service/manager_rest/cryptography_utils.py

```diff
@@ -39,3 +48,20 @@ def _get_encryption_key():
     with open(SECURITY_FILE_LOCATION) as security_conf_file:
         rest_security_conf = json.load(security_conf_file)
         return rest_security_conf['encryption_key']
+
+
+class Fernet256(Fernet):
+    def __init__(self, key, **kwargs):
+        super(Fernet256, self).__init__(Fernet.generate_key(), **kwargs)
```

Comment:
Correct- we're initialising the AES128 version of Fernet (as in: The One True Fernet), and it needs a 32 byte key or it will become very sad. However, as it doesn't do anything with this key beyond what we're about to do with our (longer) key, we can just throw this in there and let it set everything else up it needs to now (or in the future), then move on with the longer keys.

Of course, this might need revisiting if they change what they're doing in the future, but I don't think we can reliably predict what will need changing yet.